common/journal_entries/dvg_lotharingia.txt

```diff
@@ -0,0 +1,329 @@
+je_lotharian_house_of_cards = {
+	icon = "gfx/interface/icons/event_icons/event_scales.dds"
+
+	is_shown_when_inactive = {
+		exists = c:LOT
+		c:LOT = ROOT
+		NOT = { has_variable = meiji_restoration_complete }
+		has_law = law_type:law_autocracy
+	}
+
+	possible = {
+		ig:ig_armed_forces = {
+			is_in_government = yes
+		}
+	}
+
+	immediate = {
+		set_variable = {
+			name = aigle_var
+			value = 0
+		}
+	}
+	on_monthly_pulse = {
+		effect = {
+			if = {
+				limit = {
+					ig:ig_armed_forces = {
+						is_powerful = yes
+						is_in_government = yes
+					}
+					
+					
+					
+				}
+				change_variable = {
+					name = aigle_var
+					add = 1
+				}
+			}
+		}
+	}
+
+	complete = {
+		# If-else clause to avoid errors because of missing variable before JE activates
+		trigger_if = {
+			limit = { exists = var:aigle_var }
+			scope:journal_entry = { is_goal_complete = yes }
+		}
+		trigger_else = {
+			custom_tooltip = je_lotharian_collapse
+		}
+		
+		has_law = law_type:law_autocracy
+		root.average_sol <= 12
+		
+		
+	}
+	
+	on_complete = {
+		set_variable = meiji_restoration_complete
+		trigger_event = {
+			id = dvg_lotharingia.5
+		}
+	}
+	
+	fail = {
+	OR = {
+		ig:ig_armed_forces = {
+			is_powerful = no
+			is_in_government = no
+		}
+		
+		NOT = { has_law = law_type:law_autocracy				
+
+		}
+		
+		
+		
+	}
+	}
+	
+	
+	on_fail = {
+		set_variable = meiji_restoration_complete
+		trigger_event = {
+			id = dvg_lotharingia.4
+		}
+	}
+	
+	timeout = 9125
+	on_timeout = {
+		
+		trigger_event = {
+			 id = dvg_lotharingia.2
+		}
+	}
+
+	current_value = {
+		value = root.var:aigle_var
+	}
+
+	goal_add_value = {
+		value = 150
+	}
+
+	progressbar = yes
+
+	weight = 10000
+	should_be_pinned_by_default = yes
+}
+
+je_reconquering_atlantia = {
+	icon = "gfx/interface/icons/event_icons/event_military.dds"
+
+	is_shown_when_inactive = {
+		exists = c:AMA
+		NOT = {
+			owns_entire_state_region = STATE_COTE_DU_CAFE
+		}
+		any_neighbouring_state = {
+			owner = c:AMA
+		}
+		OR = {
+			country_has_primary_culture = cu:lotharian
+		}
+	}
+
+	possible = {
+	
+		NOT = {
+			is_at_war = no
+			year = 1840
+			owns_entire_state_region = STATE_COTE_DU_CAFE
+		}
+		
+		
+		
+	}
+
+	complete = {
+		OR = {
+			#DVG
+			owns_entire_state_region = STATE_COTE_DU_CAFE
+		}
+	}
+
+	on_complete = {
+		change_variable = {
+					name = aigle_var
+					add = 10
+				}
+		trigger_event = { id = reconquering_atlantia.1 }
+	}
+}
+
+je_reuniting_lotharingia = {
+	icon = "gfx/interface/icons/event_icons/event_military.dds"
+
+	is_shown_when_inactive = {
+		exists = c:LOT
+		any_neighbouring_state = {

```

Comment:
Might also want to add checks if these tags exist before checking them, to avoid errors.